### PR TITLE
Select disabled e2e features manually and enable SCTP/Seccomp tests

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
The `[Feature:SCTP]` and `[Feature:Seccomp]` are now enabled by default
as well as the lint issue got fixed.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: internal/criocli/criocli_test.go [hunk 1]
```diff
@@ -48,8 +48,8 @@ var _ = t.Describe("CLI", func() {
 
 	It("should return a copy of the slice", func() {
 		// Given
-		slice.Set("value1")
-		slice.Set("value2")
+		Expect(slice.Set("value1")).To(BeNil())
```
Comment (haircommander) [inline internal/criocli/criocli_test.go:51]: I think this change snuck in, can we have indication it's added in the pr or commit message?

**Comment by haircommander** [comment]
should these tests be run in e2e_features_* instead of e2e_*?

File: internal/criocli/criocli_test.go (hunk 1 above)
Comment (saschagrunert) [reply internal/criocli/criocli_test.go:51]: Yep, I'll add a note to fix the linter.

**Comment by saschagrunert** [comment]
> should these tests be run in e2e_features_* instead of e2e_*?

Probably, I added them to the features scoped tests.

**Comment by saschagrunert** [comment]
Huh?
/retest

**Comment by saschagrunert** [comment]
/retest

Two tests seems failing and need investigation.
/hold

**Comment by TomSweeneyRedHat** [comment]
LGTM
and happy green test buttons.

**Pull request merged**